test/stages/stage2-cp4d-instance.tf

```diff
@@ -0,0 +1,17 @@
+# module "cp4d-instance" {
```

Comment:
If this test will always be run on a cluster that has CP4D installed, then this file is not necessary and can be removed.